src/components/presentational/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -225,7 +225,7 @@ export default function TimeSeriesChart(props: TimeSeriesChartProps) {
             }
             {props.chartHasData && props.chartType === "Line" &&
                 <ResponsiveContainer width="100%" height={150}>
-                    <LineChart width={400} height={400} data={dataToDisplay} syncId="DailyDataChart" margin={{left: 5, top: 5, bottom: 5, right: 40}}>
+                    <LineChart width={400} height={400} data={dataToDisplay} syncId="DailyDataChart">
```

Comment:
> * There was some margins added to the charts so there was a large white space to the right of the chart.  I don't think these manually added margins are necessary as far as I can tell?  @ajmenca not sure if i'm missing what these accomplished.  Wondering if they were to prevent clipping of the last x-axis tick, which maybe started happening because of the chart changes (it isn't happening in current view builder branch - tick seems to align itself inside the boundaries).  Either way, I added a quick CSS tweak to make the recharts-surface have overflow:visible which prevents the clipping

This margin *is* because specifically the Intraday version of the tick label was being cut off because it's longer than our other tick labels.  There's a bunch of things that contributed to this involving how and why I ended up changing the `interval` setting on the X-Axis and specifically generating X-Axis ticks in order to get consistent and reproducible behavior out of Recharts.  I can go over all that if you want, but the long story short is that I'm confident that I wouldn't want to change any of that code.

That said - your fix doesn't fully work, as shown in the story for Intraday:

![image](https://github.com/CareEvolution/MyDataHelpsUI/assets/5473619/5aeb750f-ad12-4e7b-929d-3d16f3d9cfd1)

I agree, in general though, that the right side margin does look a little strange (and probably stranger in context of MDH rather than in the Storybook).

My suggestion is to change line 61 where the label for this is generated to this:

```
                <text fill="var(--mdhui-text-color-2)" x={x} y={y + 8} textAnchor="middle" fontSize="12">{format(currentDate, "hh:mm")}</text>
                <text fill="var(--mdhui-text-color-2)" x={x} y={y + 24} textAnchor="middle" fontSize="12">{format(currentDate, "aaa")}</text>
```

Which, in combination with removing the margin and doing your overflow css looks like this:
![image](https://github.com/CareEvolution/MyDataHelpsUI/assets/5473619/0e5da2ae-5e8c-4431-ab31-9a9fab22079d)

Those y offsets look a little tight, to be honest, so maybe its worth bumping them both down a little bit?  They're copied from the offsets that we set on the two line label if you're in the Week view, which could also maybe use a bit of extra breathing room?
![image](https://github.com/CareEvolution/MyDataHelpsUI/assets/5473619/050cbb2d-ff65-4157-94ff-8a965018aba4)

Reply:
Yeah they are a bit tight, i'll bump them down a couple px